lk2nd/lk2nd-device.c

```diff
@@ -273,15 +274,33 @@ static void lk2nd_parse_panels(const void *fdt, int offset)
 	}
 
 	new = lkfdt_getprop_str(fdt, offset, "compatible", &new_len);
-	if (!new || new_len < 1)
-		return;
+	if (!new || new_len < 1) {
+		new = lkfdt_getprop_str(fdt, offset, "replace-compatible", &new_len);
+		if (!new || new_len < 1)
+			return;
+		replace_mode = true;
+		dprintf(INFO, "Panel compatible will be *replaced*\n");
```

Comment:
not sure if this message adds much value

Reply:
Yes, it was a debug print and plan was to remove it anyway

Reply:
fixed